### DISCUSSION TITLE
Notifications Inbox Unusable due to Orgs

### DISCUSSION THREAD

**Discussion by acook** [community/community#17478] [Other Feature Feedback, Questions, & Ideas]
Being unable to filter out certain orgs I'm part of is a huge UX-breaking issue. My notifications page will be blank if an org I'm part of has a bunch of notifications but I'm not SSO'd into it. I don't SSO into company resources on my personal machine. 

This means that the notifications inbox is fundamentally broken and I have missed contributors on my personal repos for weeks or months as a result of them being buried.

There are janky workarounds, but even basic `NOT` or `-` prefix doesn't work on the inbox filter to exclude the org, so I have to make a custom filter which I have to manually maintain an allowlist.

There is no way to remove or hide an SSO'd org's notifications other than logging into the org or leaving it.

Clearly, we need improvements to the UX here, it's been years. Please fix this.

Other than having an option to completely hide orgs, at least letting `-org` or `NOT org` would be a great help in the meantime.

**Comment by BenjamenMeyer** [comment]
I have found myself saving filters for the different orgs I work with which lets me focus directly on those; the only issue I have there is not being able to then group-by repository so it's harder to sub-filter/split the work into the repos I care more about and those I care less about.

I do agree having a `NOT` expression in the filter would be useful especially as https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/viewing-and-triaging-notifications/managing-notifications-from-your-inbox#custom-filter-limitations doesn't discuss limits on its usage; you just get warnings about it in the filter.
